console/style.rst

```diff
@@ -238,7 +238,7 @@ User Input Methods
     the third argument::
 
         $io->ask('Number of workers to start', 1, function ($number) {
-            if (!is_integer($number)) {
+            if (!ctype_digit($number)) {
```

Comment:
Won't this fail if the user types nothing and the default value (which actually is an integer) is used? What about combining both checks?

Reply:
Could we use http://php.net/is_numeric first and then cast to int?